io-engine/src/bdev/nexus/nexus_child.rs

```diff
@@ -1232,4 +1237,37 @@ impl<'c> NexusChild<'c> {
             }
         }
     }
+
+    /// Creates a new rebuild log, or returns an existing one.
+    pub(super) fn get_or_init_rebuild_log(
+        &mut self,
+    ) -> Option<RebuildLogHandle> {
```

Comment:
Shouldn't this return `RebuildLogHandle` rather than Option since it's `get_or_init`?

Reply:
I would like to have rebuild log optional.
E.g. we can disable it completely by just returning None here, without modifying other places. Should probably add an env var or something to support that.
Also, potentially, we may add logic when a child refuses to create a rebuild log, e.g. if avail memory less than the log size would be; or for debugging purposes.